src/govuk/components/header/template.njk

```diff
@@ -39,7 +39,7 @@
 
```

Comment:
It would be good to correct the above comment to say 
> an empty data URI xlink:href

instead of
> empty xlink:href attribute

Reply:
Hi @hannalaakso I've made the changes you suggested. Thanks!